docs/content/Getting-Started/Cloud/03-Generate-models.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Generate models
+permalink: /getting-started/cloud/generate-models
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 4
+isDisableFeedbackBlock: true
+---
+
+## Select tables
+
+Start by selecting the database tables to generate the data schema from, then
+click **Measures and Dimensions**:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Generate Models Screen detected tables from data source for selection"
+    src="https://ucarecdn.com/e0420b86-64bc-4e01-aa74-16a7e926f523/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Measures and dimensions
+
+Next, check the detected measures and dimensions to ensure correctness. After
```

Comment:
Sounds a bit strange: if the sample dataset is used, it should work out of the box; if a reader uses her own dataset, what should she look at, exactly? What can be incorrect?